packages/create-cloudflare/src/__tests__/dialog.test.ts

```diff
@@ -15,33 +15,69 @@ describe("dialog helpers", () => {
 		process.stdout.columns = originalColumns;
 	});
 
-	test("printWelcomeMessage with telemetry disabled", () => {
-		printWelcomeMessage("0.0.0", false);
+	describe("printWelcomeMessage", () => {
+		test("with telemetry disabled", () => {
+			printWelcomeMessage("0.0.0", false, {});
 
-		expect(normalizeOutput(std.out)).toMatchInlineSnapshot(`
-			"────────────────────────────────────────────────────────────
-			👋 Welcome to create-cloudflare v0.0.0!
-			🧡 Let's get started.
-			────────────────────────────────────────────────────────────
+			expect(normalizeOutput(std.out)).toMatchInlineSnapshot(`
+				"────────────────────────────────────────────────────────────
+				👋 Welcome to create-cloudflare v0.0.0!
+				🧡 Let's get started.
+				────────────────────────────────────────────────────────────
 
-			"
-		`);
-	});
+				"
+			`);
+		});
+
+		test("with telemetry enabled", () => {
+			printWelcomeMessage("0.0.0", true, {});
+
+			expect(normalizeOutput(std.out)).toMatchInlineSnapshot(`
+				"────────────────────────────────────────────────────────────
+				👋 Welcome to create-cloudflare v0.0.0!
+				🧡 Let's get started.
+				📊 Cloudflare collects telemetry about your usage of Create-Cloudflare.
```

Comment:
Not a big deal but just chiming in here after the fact 😄 

In the case that it is not experimental mode there is no empty line between "let's get started" and the telemetry line. So adding extra lines when in experimental mode is a change of look.